database/sqlite/sqlite_health.c

```diff
@@ -1819,6 +1819,14 @@ void sql_health_alarm_log2json_v3(BUFFER *wb, DICTIONARY *alert_instances, time_
             if (unlikely(rc != SQLITE_OK))
                 error_report("Failed to bind aci parameter.");
 
+            rc = sqlite3_bind_int(res, 8, (int) t->ni);
```

Comment:
@ktsaou in normal situation when we cannot bind a variable, instead to proceed and try to bind the next, we are reporting an error and stopping, because the transaction can fail, please, take a look [here](https://github.com/thiagoftsm/netdata/blob/52ac7b0f1a2de17feeb2613d2a32a6aa17429cbc/database/sqlite/sqlite_aclk_alert.c#L35-L39).

Reply:
this is for @stelfrag

Reply:
Added check